datamodel/src/main/java/com/hartwig/serve/datamodel/serialization/util/ActionableFileUtil.java

```diff
@@ -132,6 +134,10 @@ public EvidenceLevel level() {
                 return EvidenceLevel.valueOf(values[fields.get("level")]);
             }
 
+            @NotNull
+            @Override
+            public ApprovalStatus approvalStatus() { return ApprovalStatus.fromString(values[fields.get("approvalStatus")]); }
```

Comment:
I think it's more in line with how we write the rest of the output file, to write the ENUM value to the output file (e.g. CASE_REPORTS_SERIES instead of Case Reports/Case series)